doc/requirements.md

```diff
@@ -37,9 +37,11 @@ The library must handle out-of-memory conditions gracefully in a variety of situ
 
 The operations identified in this section should be declared in a separate C++ namespace (pelopia); if they are implemented as methods on a class, so should be the class.
 
-In the operation descriptions, the notation { X } is used to denote a possibly empty collection of objects of type X.  
+In the operations' descriptions below, the notation { X } is used to denote a possibly empty collection of objects of type X.  
 
-###Search ( string, categories, options ) : { string }
+The result sets (**Geocode**) returned by the search operations should be represented in conformance to [geocodejson-spec](https://github.com/geocoders/geocodejson-spec). In the context of a C++ API, this means a class providing read access to all properties prescribed by the specification, as well as a function/method to convert the object into a character string representing an equivalent GeocodeJSON object.
```

Comment:
Call this part **Response** instead of **Geocode** since geocode is a verb, not noun.